### PR TITLE
[WIP] Prepping to Migrate Reports to Bootstrap 3

### PR DESCRIPTION
**Pull request by biyeun**
Moving files around and duplicating for b3 in preparation to migrate reports to bootstrap 3

@sravfeyn cc @calellowitz @benrudolph

### CODE REVIEW

File: corehq/apps/reports/templates/reports/bootstrap3/base_template.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{% extends "reports/bootstrap3/standard/base_template.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% block js %}{{ block.super }}
+    <script src="{% static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
```
Comment (sravfeyn) [inline corehq/apps/reports/templates/reports/bootstrap3/base_template.html]: Tests are expecting this to be `newstatic`

**Comment by benrudolph** [comment]
assuming most of the code is just moved. seems fine once tests pass. if there's something more complicated going on let me know and we can go over together?

**Comment by biyeun** [comment]
@benrudolph yep you are correct

**Comment by biyeun** [comment]
@benrudolph tests pass

**Pull request merged**